projects/fsastorefrontlib/src/cms-components/checkout/components/configure-product/mini-cart/product-configuration-mini-cart.component.ts

```diff
@@ -82,6 +83,16 @@ export class ProductConfigurationMiniCartComponent
     );
   }
 
+  getPaynowFormattedPrice(oneTimeChargeEntries: any): string {
+    let paynowFormattedPrice: string;
+    oneTimeChargeEntries.forEach(oneTimeChargeEntry => {
+      if (oneTimeChargeEntry.billingTime.code === this.paynowBillingTimeCode) {
```

Comment:
Why not just compare it to 'paynow', since `paynowBillingTimeCode` is readonly and private so it cannot be changed, right?